importer/make_KMB_info.py

```diff
@@ -465,6 +465,10 @@ def generate_content_cats(self, item):
         :param item: the KMBItem to analyse
         :return: list of categories (without "Category:" prefix)
         """
+        # Add parish/municipality categorisation when needed
```

Comment:
these need to be after both `# depicted` and  `# add tag categories unless a commonscat was found` as they modify `item.needs_place_cat`.

Reply:
Will this not re-add categories that might have been removed as part of exact category adding?

Reply:
What I'm suggesting is the below. I.e. exact match is the last to be called since it relies on both class, keyword and place categories (if appropriate) for parent category matching. And since it's in the end nothing it removes will be re-added.

```python
# depicted
found_commonscat = item.make_commonscat_categories(self.category_cache)

# add tag categories unless a commonscat was found
if not found_commonscat:
    item.make_item_class_categories(self.category_cache)
    item.make_item_keyword_categories(self.category_cache)

# Add parish/municipality categorisation when needed
if item.needs_place_cat:
    item.make_place_category()

if not found_commonscat:
    item.get_exact_cat_from_name(self.category_cache)

return list(item.content_cats)
```

Reply:
Done.